src/app/(main)/my-voyage/[teamId]/directory/components/DirectoryComponentWrapper.tsx

```diff
@@ -0,0 +1,135 @@
+import { redirect } from "next/navigation";
+import DirectoryProvider from "./DirectoryProvider";
+import TeamMember from "./TeamMember";
+import { TeamDirectory } from "@/store/features/directory/directorySlice";
+import Banner from "@/components/banner/Banner";
+import { getAccessToken } from "@/utils/getCookie";
+import { AsyncActionResponse, handleAsync } from "@/utils/handleAsync";
+import { GET } from "@/utils/requests";
+import { CacheTag } from "@/utils/cacheTag";
+import { User, VoyageTeamMember } from "@/store/features/user/userSlice";
+import { getUser } from "@/utils/getUser";
+import { getTimezone } from "@/utils/getTimezone";
+import VoyagePageBannerContainer from "@/components/banner/VoyagePageBannerContainer";
+
+interface FetchTeamDirectoryProps {
+  teamId: number;
+  user: User | null;
+}
+
+export async function fetchTeamDirectory({
```

Comment:
@Dan-Y-Ko Why don't we have GET requests in `...Service.ts` too? Am I missing smth?